shared/src/commonMain/kotlin/co/nimblehq/ic/kmm/suv/data/remote/model/QuestionApiModel.kt

```diff
@@ -27,9 +26,7 @@ fun QuestionApiModel.toQuestion(): Question = Question(
     id = id,
     text = text,
     displayOrder = displayOrder,
-    displayType =
-    QuestionDisplayType.values().firstOrNull { displayType == it.value }
-        ?: QuestionDisplayType.UNSUPPORTED,
```

Comment:
With these changes, we don't need to make use of `sealed class` and can revert back to the simple converting by using an enum 🤓 

What do you think?